### DISCUSSION TITLE
Polling metadata from Liquidsoap/Azuracast

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#4877] [Q&A] [answered]
On a Shoutcast server, it's possible to poll the song metadata at this addy [http(s)://(ip):(channel)/7.html] and there's a similar link in Ice as well. Is there a way to poll the metadata directly from Azura/Liquidsoap?

**Comment by BusterNeece** [comment, marked as answer]
We don't have anything that returns a single line a-la SHOUTcast's legacy `7.html`, but we do have a robust API that's available for all stations that returns the current playing track, any rich metadata associated with it, and other info about the station and its schedule: https://docs.azuracast.com/en/developers/apis/now-playing-data

**Reply by SpeedyCharly** [reply]
Danke mein Herr!
